cmd/retransmitter/retransmit/network/incoming_peer.go

```diff
@@ -66,7 +66,7 @@ func RunIncomingPeer(ctx context.Context, params IncomingPeerParams) {
 	writeHandshake := proto.Handshake{
 		AppName: params.WavesNetwork,
 		// pass the same minor version as received
-		Version:      proto.Version{Major: 0, Minor: readHandshake.Version.Minor, Patch: 0},
```

Comment:
Looks very strange. Why we always pass 0 as major version? I think we should only drop patch part from the incoming version.

Reply:
I agree. Fixed